plugins/woocommerce/src/Admin/FeaturePlugin.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * WooCommerce Admin: Feature plugin main class.
+ */
+
+namespace Automattic\WooCommerce\Admin;
+
+defined( 'ABSPATH' ) || exit;
+
+
+/**
+ * Feature plugin main class.
+ *
+ * @internal This file will not be bundled with woo core, only the feature plugin.
+ * @internal Note this is not called WC_Admin due to a class already existing in core with that name.
+ */
+class FeaturePlugin extends DeprecatedClassFacade {
```

Comment:
Getting this error on the front-end (non-admin) side of the site:
<img width="1187" alt="Screen Shot 2022-05-11 at 1 19 23 PM" src="https://user-images.githubusercontent.com/10561050/167909284-22b52541-13c4-4202-8d8a-0446bbd91702.png">